src/main/java/africa/vote/SmartVote/controllers/SampleController.java

```diff
@@ -14,7 +14,7 @@
 @RestController
 @CrossOrigin("*")
```

Comment:
Ogbeni, Go and learn how to implement a proper CORS filter... you dont want to open cross origin requests to just anyone in an enterprise environment